validation/models.py

```diff
@@ -353,6 +355,16 @@ def generate_primary_key(sender, instance, **kwargs):
 SHA256_HEX_LENGTH = 64
 
 
+class Status(models.Model):
```

Comment:
I would actually make this a [OneToOneField](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.OneToOneField) that references the recording HERE, rather than having the recording reference the issue.

i.e.,

```suggestion
class Status(models.Model):
    recording = models.OneToOneField("Recording", on_delete=models.SET_NULL)
```

(then delete the `ForeignKey` in Recording).

This states that a `Status` **must** be associated with exactly one `Recording`. Otherwise, the schema technically allows for "floating" `Status` records that are not associated with any recording.

Django then will do some ORM magic such that accessing `recording.status` either gives you `None` or _exactly_ one `Status` object!

Reply:
Addendum: I was wrong about the `on_delete` field; it should be `CASCADE` so that when (if?) a recording gets deleted, its status will be deleted—no stranded data!

```python
class Status(models.Model):
    recording = models.OneToOneField("Recording", on_delete=models.CASCADE)
```

Reply:
Addendum addendum:

`recording.status` will give you a `RelatedObjectDoesNotExist` error instead of None :/